tools/gencert/ca/ca_sign.sh

```diff
@@ -5,7 +5,7 @@
 openssl req -new -key "$1" -out "$1".csr
 
 # CA sign the request
-openssl x509 -req -in "$1".csr -CA "$2" -CAkey "$3" -out "$4" -set_serial 0
+openssl x509 -days 1825 -req -in "$1".csr -CA "$2" -CAkey "$3" -out "$4" -set_serial 0
```

Comment:
Why such a short expiration for the attestation certificate? Five years is not a very long time. I'd personally go for 20+ years. If the attestation root key is compromised, then presumably you'd revoke it, not wait for expiration. Some services might reject expired attestation certificates. For example, Yubico's U2F attestation certificates expire in 2050.